templates/docs/examples/patterns/badge/tabs.html

```diff
@@ -0,0 +1,32 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Badges / Tabs{% endblock %}
+
+{% block standalone_css %}patterns_badge{% endblock %}
+
+{% block content %}
+<nav class="p-tabs" aria-label="Tabs example">
+  <ul class="p-tabs__list">
+    <li class="p-tabs__item">
+      <a href="/docs" class="p-tabs__link" aria-selected="true">Docs
```

Comment:
Don't link to any real URLs please to avoid loading them in embedded examples. These should all be `#`.
```suggestion
      <a href="#" class="p-tabs__link" aria-selected="true">Docs
```